### PR TITLE
Release 2.0.0 (develop)

### PR DESCRIPTION
**Pull request by arebra**
Version 2.0.0

Launched new developer portal theme:
*   New design
*   New menu structure
*   Reading time

Other changes:
*   Added GDPR disclaimer.
*   Added custom 404 page.
*   Added Transaction on File section.
*   Added Seamless View for Trustly.
*   Added documentation about payment states in Card Other Features.
*   Added event overrides warning for Card Seamless View Events.
*   Added section regarding MOTO in Card Other Features.
*   Deleted Merchant Identified Payer in Checkout
*   Renamed Merchant Authenticated Consumer to Delegated Strong Consumer Authentication
*   Updated `payer`/`consumer`/`end-user` naming for most sections.
*   Updated expiry date for test cards in test data.
*   Updated callback documentation.
*   Updated regex pattern for `orderItems.class`.

### CODE REVIEW

File: resources/release-notes.md [hunk 1]
```diff
@@ -10,6 +10,34 @@ description: |
...
+
+:tada: Launched new developer portal theme: :tada:
+*   New design :sparkles:
+*   New menu structure :orange_book:
+*   Reading time :hourglass:
+
+Other changes:
+*   Added GDPR disclaimer.
+*   Added custom 404 page.
+*   Added Transaction on File section.
```
Comment (asbjornu) [inline resources/release-notes.md:26]: Can we get a link to transaction on file here?

File: resources/release-notes.md [hunk 2]
```diff
@@ -10,6 +10,34 @@ description: |
...
+*   Added Seamless View for [Trustly][trustly-payments].
+*   Added documentation about payment states in
+    [Card Other Features][card-transaction-states]
+*   Added event overrides warning for [Card Seamless View Events][card-seamless-view]
```
Comment (asbjornu) [inline resources/release-notes.md:30]: Doesn't the warning apply to all seamless view events (not just for Card)?

File: resources/release-notes.md [hunk 3]
```diff
@@ -10,6 +10,34 @@ description: |
...
+*   New menu structure :orange_book:
+*   Reading time :hourglass:
+
+Other changes:
+*   Added GDPR disclaimer.
+*   Added custom 404 page.
+*   Added Transaction on File section.
+*   Added Seamless View for [Trustly][trustly-payments].
+*   Added documentation about payment states in
+    [Card Other Features][card-transaction-states]
```
Comment (asbjornu) [inline resources/release-notes.md:29]: Doesn't the payment state documentation apply to all APIs, not just Card?

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: resources/release-notes.md (hunk 2 above)
Comment (arebra) [reply resources/release-notes.md:30]: Yes it does. Is it better to leave it without a link?

File: resources/release-notes.md (hunk 3 above)
Comment (arebra) [reply resources/release-notes.md:29]: Yes it does. Is it better to leave it without a link? We added one link as an example.

File: resources/release-notes.md (hunk 1 above)
Comment (arebra) [reply resources/release-notes.md:26]: Oops! Sure!

Comment (arebra) [inline resources/release-notes.md:26]:
```suggestion
*   Added section about [Transaction on File][transaction-on-file].
```

File: resources/release-notes.md [hunk 4]
```diff
@@ -10,6 +10,34 @@ description: |
...
+*   Reading time :hourglass:
+
+Other changes:
+*   Added GDPR disclaimer.
+*   Added custom 404 page.
+*   Added section about [Transaction on File][transaction-on-file]
+*   Added Seamless View for [Trustly][trustly-payments].
+*   Added documentation about payment states in
+    [Card Other Features][card-transaction-states]
+*   Added event overrides warning for [Card Seamless View Events][card-seamless-view]
```
Comment (arebra) [inline resources/release-notes.md:30]:
```suggestion
*   Added event overrides warning for Seamless View Events.
```

File: resources/release-notes.md [hunk 5]
```diff
@@ -10,6 +10,34 @@ description: |
...
+*   New design :sparkles:
+*   New menu structure :orange_book:
+*   Reading time :hourglass:
+
+Other changes:
+*   Added GDPR disclaimer.
+*   Added custom 404 page.
+*   Added section about [Transaction on File][transaction-on-file]
+*   Added Seamless View for [Trustly][trustly-payments].
+*   Added documentation about payment states in
```
Comment (arebra) [inline resources/release-notes.md:28]:
```suggestion
*   Added documentation about payment and transaction states.
```

File: resources/release-notes.md (hunk 2 above)
Comment (asbjornu) [reply resources/release-notes.md:29]: No, but we can perhaps word it differently? Something like:

```suggestion
*   Added event override warning for Seamless View Events, see [Card Seamless View Events][card-seamless-view] for example.
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: resources/release-notes.md [hunk 6]
```diff
@@ -10,6 +10,33 @@ description: |
...
+*   New design :sparkles:
+*   New menu structure :orange_book:
+*   Reading time :hourglass:
+
+Other changes:
+*   Added GDPR disclaimer.
+*   Added custom 404 page.
+*   Added section about [Transaction on File][transaction-on-file]
+*   Added Seamless View for [Trustly][trustly-payments].
+*   Added documentation about payment and transaction states.
```
Comment (asbjornu) [inline resources/release-notes.md:28]:
```suggestion
*   Added documentation about payment and transaction states. See [Card Other Features][card-transaction-states] for example.
```

**Review by asbjornu — CHANGES_REQUESTED** [review]
1. We need to describe the release on the front page.
2. All the links on the front page needs to go somewhere (other than back to the front page).

File: index.md [hunk 7]
```diff
@@ -43,17 +43,17 @@ hide_from_sidebar: true
...
           type='sdk'
-          to=''
+          to='/modules-sdks/mobile-sdk/android'
       %}
     </div>
     <div class="{{ card_col_class }}">
       {% include card.html
-          title='Swift SDK'
+          title='iOS SDK'
           text='Learn more about how to integrate our Swift SDK'
```
Comment (asbjornu) [inline index.md:52]:
```suggestion
          text='Learn more about how to integrate our iOS SDK'
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Review by asbjornu — APPROVED** [review]

**Review by NullableInt — APPROVED** [review]

**Pull request merged**